src/main/java/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/abstractItems/VanillaContainer.java

```diff
@@ -0,0 +1,17 @@
+package me.mrCookieSlime.Slimefun.Objects.SlimefunItem.abstractItems;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.inventory.ItemStack;
+
+import me.mrCookieSlime.Slimefun.Lists.RecipeType;
+import me.mrCookieSlime.Slimefun.Objects.Category;
+import me.mrCookieSlime.Slimefun.api.SlimefunItemStack;
+
+public abstract class VanillaContainer extends AContainer {
```

Comment:
Missing javadocs, also not the best name. The name could imply different stuff and it's even worse without the docs.